SingleClientVersion.lua

```diff
@@ -10,26 +10,25 @@
 --- The only allowed client version number
 local g_AllowedVersion
 
-
-
-
-
---- Maps version name ("1.7.10") to version number (5)
```

Comment:
Why did you remove the empty lines? We separate major parts of the code visually, to stand out more. Please keep these.